doc/api_assets/style.css

```diff
@@ -207,6 +213,12 @@ header h1 {
   padding-top: 1em;
 }
 
+#apicontent .line {
+  width: calc(50% - 1em);
```

Comment:
nit: calc is discouraged for portability and preformance, no? If copy paste then defer this for some later time.